sidekiq/README.md

```diff
@@ -0,0 +1,144 @@
+# Agent Integration: Sidekiq
+
+## Overview
+
+This integration monitors [Sidekiq][3] through [Dogstatsd][5]. It collects metrics through [Datadog's Dogstatsd Ruby client][4].
+
+**Note** Only Sidekiq Pro (>= 3.6) or Enterprise (>= 1.1.0) users can collect metrics.
+
+## Setup
+
+### Installation
+
+The Sidekiq integration is packaged with the [Datadog Agent][2].
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Install the `dogstatsd-ruby` [gem][4]:
+
+   ```
+    gem install dogstatsd-ruby
+   ```
+
+2. Enable Sidekiq Pro metric collection by including this in your initializer:
+
+   ```ruby
+        require 'datadog/statsd' # gem 'dogstatsd-ruby'
+
+        Sidekiq::Pro.dogstatsd = ->{ Datadog::Statsd.new('metrics.example.com', 8125, namespace:'sidekiq') }
+
+        Sidekiq.configure_server do |config|
+          config.server_middleware do |chain|
+            require 'sidekiq/middleware/server/statsd'
+            chain.add Sidekiq::Middleware::Server::Statsd
+          end
+        end
+   ```
+
+    If you are using Sidekiq Enterprise and would like to collect historical metrics, include this line as well:
+
+   ```ruby
+          Sidekiq.configure_server do |config|
+            # history is captured every 30 seconds by default
+            config.retain_history(30)
+          end
+   ```
+
+  See the Sidekiq [Pro][6] and [Enterprise][7] documentation for more information, and the [Datadog Ruby][7] documentation for further configuration options.
+
+4. Update the [Datadog Agent main configuration file][13] `datadog.yaml` by adding the following configs:
```

Comment:
```suggestion
3. Update the [Datadog Agent main configuration file][13] `datadog.yaml` by adding the following configs:
```

Reply:
Step 5 should also be changed to step 4.

Reply:
Sure!